openy_af4_vue_app/src/components/ResultsBar.vue

```diff
@@ -86,7 +86,7 @@ export default {
 </script>
 
 <style lang="scss">
-.results-bar-component {
+.results-bar-component .container {
```

Comment:
Why do we need this change? If .container will have margins - then the background won't fill the whole width of the component.

Reply:
We had a small style bug on tablet https://i.imgur.com/YJyHTEL.png and after this change it looks https://i.imgur.com/ryGSscV.png

Reply:
Please remove this modification. It is not related to this issue.
And it should look like this on tablet:
![image](https://user-images.githubusercontent.com/2128648/113400285-20f1cf00-93d4-11eb-8245-5dc7d47ce09f.png)
the gray area should fill all the space from left to right.
The bug you referenced should probably be fixed in another way - using another page template for the landing page... or smth else.

Reply:
Reverted.